### ISSUE TITLE
Clear instructions for setting up everything upfront & using PowerShell on Windows

### ISSUE THREAD

**Issue by hmgaudecker** [hmgaudecker/econ-project-templates#63]
- [ ] Insert a step 2 to make sure everybody has exactly the same setup: 
   - [x] Terminal on Mac / Linux, Powershell on Windows. 
   - [x] In correct location. No spaces / special characters in path (see prog-econ repo, slides on Shell Introduction). 
    - [x] All paths to required programmes should be correct (explain that we will need to launch them from the shell), i.e., `pdflatex` and, if required, anything that is required like `stata`, `matlab`, `r` should start the respective programme (check the files `{{cookiecutter.project_slug}}/.mywaflib/waflib/extras/run_*_script.py`). Assume atom is installed and `atom` launches editor from shell. Similar for `git`.  
    - [x] Links to how to set paths on MacOS, Windows. Explain that new shell needs to be launched./ 
- [ ] New step 3 (currently 2) should read 
       ```
$ conda deactivate
$ conda install cookiecutter pre-commit
      ```
   First part just to be sure. Second because conda init often fails (Mac, Windows) for reasons I have not fully understood.
- [x] Add a step 7 (currently between 5 and 6) on what to do if anything fails: 
      1. Understand the error
      2. Type 
          ```
$ atom ~/.cookiecutter_replay/econ-project-template-[version].json
          ```
          Fix the faulty setting if something went wrong there, alternatively fix paths etc.  When done, launch a new shell if necessary and type:
          ```
$ cookiecutter --replay https://github.com/hmgaudecker/econ-project-templates/archive/[version].zip
          ```

**Comment by raholler** [comment]
is that with the paths only a windows problem or also a mac problem?

**Comment by raholler** [comment]
Regarding the point with the pre-commits, I think the cleanest wa to do it and this also has a "Lerneffekt" for the pre-commit users, is to do it in step 7 after activate env , run pre-commit install... because that is what you always have to do if you want to use pre-commmit hooks outside of the template. And then people dont have to put it in the base env if they dont want to.

**Comment by raholler** [comment]
what is the terminal in mac called?

**Comment by hmgaudecker** [comment]
> is that with the paths only a windows problem or also a mac problem?

Well, a "problem" in the sense that system paths are not modified by programmes that are installed, which is mostly a good thing. The only actual problem is that most new users have no idea where to modify them. Same on Linux -- unless you install stuff via package managers -- but there we can assume people know what they are doing.

On MacOS, it is just a matter of modifying things in `~/.bash_profiile`, the instructions in the econ-python-environment docs on github.io should be reasonably up-to-date.

> what is the terminal in mac called?

Terminal.

**Comment by hmgaudecker** [comment]
> Regarding the point with the pre-commits, I think the cleanest wa to do it and this also has a "Lerneffekt" for the pre-commit users, is to do it in step 7 after activate env , run pre-commit install... because that is what you always have to do if you want to use pre-commmit hooks outside of the template. And then people dont have to put it in the base env if they dont want to.

Sounds good.

**Comment by raholler** [comment]
Does the atom refer to the editor? Not everyone might have this.

**Comment by hmgaudecker** [comment]
Yes. We should include it in the installation instructions. If somebody wants to use something else, they should know how to start whatever they want to use.

**Comment by raholler** [comment]
fixed in #65
